eve/main.yml

```diff
@@ -133,12 +133,29 @@ stages:
               workloadPlane: 10.0.0.0/8
             END
             EOF
+      - ShellCommand:
+          # This step is needed so Kubelet registration doesn't fail for DNS
+          # entries longer than 63 characters
+          name: Change hostname to a shorter value
+          command: sudo hostnamectl set-hostname bootstrap
       - ShellCommand:
           name: Start the bootstrap process
           command: >
             sudo bash
             /srv/scality/metalk8s-%(prop:metalk8s_short_version)s/bootstrap.sh
           haltOnFailure: true
+      - ShellCommand:
+          name: Install kubectl for running tests
+          command: >
+            sudo yum install -y kubectl --disablerepo=*
+            --enablerepo=metalk8s-kubernetes
+      - ShellCommand:
+          name: Run tests on the bootstrap node
```

Comment:
minor: remove `on the bootstrap node` since here we are on a worker that works a bootstrap node. That may be misleading.

Reply:
Tracked [here](https://github.com/scality/metalk8s/issues/781#issuecomment-476565028).